components/builder-web/app/util.ts

```diff
@@ -118,32 +118,32 @@ export function isEarlyAccess() {
 export const targets = [
   {
     id: 'x86_64-linux',
-    name: 'Linux',
-    title: 'Linux',
+    name: 'Linux (x86_64)',
+    title: 'Linux (x86_64)',
     param: 'linux'
   },
   {
```

Comment:
We should add an entry for `aarch64-linux`.

Reply:
Added an entry in line with the others that I think should work.  You'll see it when I push again.

Reply:
Is there a reason why this is `aarch64` and the M1 is `ARM64`?

Reply:
I went with ARM64 in the M1 case because that's I believe that is what Apple uses so I went with that there. Based on previously looking at things like [Rust's platform support docs](https://doc.rust-lang.org/nightly/rustc/platform-support.html) I thought aarch64 was the commonly used term pretty much everywhere else.  Your question made me go dig again and based on [this Raymond Chen blog](https://devblogs.microsoft.com/oldnewthing/20220726-00/?p=106898) I guess Windows actually goes with ARM64 too? I'm certainly not gonna argue with Raymond Chen anyway.  So the idea was to use the nomenclature that I thought a given community used.  If I were going to change something I would probably break this way but I'm really tempted to keep the 'macOS (ARM64)' as I think that's what's common in Apple circles.

```
{
  id: 'x86_64-darwin',
  name: 'macOS Intel',
  title: 'macOS (x86_64)',
  param: 'mac'
},
{
  id: 'aarch64-darwin',
  name: 'macOS M1',
  title: 'macOS (aarch64)',
  param: 'mac_m1'
}
```

Reply:
ok if arm64 is ubiquitous for apple then that's cool.

Reply:
Yeah, I think it is.  I went and did some more digging online and everywhere the architecture seems to be discussed ARM64 is being used.  However, in thinking about this I do think I'm going to make the mac entries match what's in the above comment which I've edited since the original post.  I think those entries would best reflect what's in commonly used based on just looking around a bit.